### PR TITLE
Remove SUMO from Webots package

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
As discussed in  #6232, remove SUMO from the Webots package and document how to install it on the system.

### CODE REVIEW

File: projects/default/controllers/sumo_supervisor/sumo_supervisor.py [hunk 1]
```diff
@@ -24,22 +24,28 @@
...
-        sumoPath = os.path.join(sumoPath, 'projects', 'default', 'resources', 'sumo')
-        os.putenv("SUMO_HOME", sumoPath)
+        sumoImportError()
+
     if sys.platform.startswith('darwin'):
         libraryVariablePath = 'DYLD_LIBRARY_PATH'
-
         # Fix the following error (shown on "El Capitan"):
         #    ./projects/default/resources/sumo/sumo-gui
         #    Fontconfig error: Cannot load default config file
```
Comment (stefaniapedrazzi) [inline projects/default/controllers/sumo_supervisor/sumo_supervisor.py:51]: It should be checked if this is still needed on macOS

**Comment by stefaniapedrazzi** [comment]
I completed to remove the sumo files from the Webots package and development system, as well as updated the documentation to refer to the system SUMO installation.

Instructions to install SUMO are currently in these two files:
* projects/default/controllers/sumo_supervisor/sumo_supervisor.py
* docs/automobile/sumo-interface.md

Currently I tested the changes only on Linux.
We should also test them on macOS and Windows.

**Comment by stefaniapedrazzi** [comment]
@ygoumaz could you please check the changes for macOS?

File: projects/default/controllers/sumo_supervisor/sumo_supervisor.py (hunk 1 above)
Comment (ygoumaz) [reply projects/default/controllers/sumo_supervisor/sumo_supervisor.py:51]: It works on macOS without those lines.

**Comment by omichel** [comment]
I tested on Windows 11 and it works like a charm.

**Comment by ygoumaz** [comment]
I have tested on all platforms and it works nicely. It is very easy to install and use the SUMO package, while decreasing the size of the Webots one.

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**